pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -201,6 +201,30 @@ var adminCommands = []*cli.Command{
 			{
 				Name:  "conflicts",
 				Usage: "runs a conflict resolution to find users with multiple entries",
+				CustomHelpTemplate: `
```

Comment:
@IevaVasiljeva fyi, found a "customhelp template" field in the /urfave library that we are using.

Might be interesting for people in the future to add more help messages for the CLI commands 👍